x-pack/plugins/security_solution/server/lib/telemetry/helpers.ts

```diff
@@ -300,3 +301,46 @@ export const createTaskMetric = (
     error_message: errorMessage,
   };
 };
+
+function obfuscateString(clusterId: string, toHash: string): string {
+  const valueToObfuscate = toHash + clusterId;
+  return sha256.create().update(valueToObfuscate).hex();
+}
+
+function isAllowlistK8sUsername(username: string) {
```

Comment:
should elastic-agent be included here as per comment: https://github.com/elastic/kibana/pull/152129#discussion_r1127214042